src/accelerate/utils/dataclasses.py

```diff
@@ -1338,7 +1338,7 @@ class FullyShardedDataParallelPlugin:
         },
     )
     sync_module_states: bool = field(
-        default=False,
+        default=None,
         metadata={
             "help": "Whether each individually wrapped FSDP unit should broadcast module parameters from rank 0 "
             "to ensure they are the same across all ranks after initialization. Defaults to `True`"
```

Comment:
In the post_init, I can see that actually the default is `False`. Either change the description or the init below. Also thanks for fixing this !